subprojects/docs/src/samples/workerApi/noIsolation/groovy/build.gradle

```diff
@@ -19,14 +19,25 @@ import javax.inject.Inject
 // The parameters for a single unit of work
 interface ReverseParameters extends WorkParameters {
     Property<File> getFileToReverse()
-    Property<File> getDestinationFile()
+    Property<File> getDestinationDir()
```

Comment:
Can we fix the samples to use `RegularFileProperty`/`DirectoryProperty` vs `Provider<File>`?

Reply:
Yep, done.